.github/workflows/auto-review.yaml

```diff
@@ -0,0 +1,42 @@
+name: AutoReview
+
+on:
+    push:
+        branches: [ main, master ]
```

Comment:
Could you explain why the `main` branch is added in the `branches` setting?

Reply:
Currently, there is no reason.

I would like however to rename `master` to `main` at some point, allowing the two will allow to do that any time without any issue. Note that the other added workflow have the same too